<?xml version="1.0" encoding="utf-8"?>

<launch>
	
	<!-- VIEW THE RAW IMAGE -->
	<!--<node pkg="image_view" type="image_view" name="image_view" output="screen">
		<remap from="image" to="camera/rgb/image_rect_color" />
		<param name="autosize" type="bool" value="true" />
	</node>-->


	<!-- AR NODE - DETECTING THE BOARD -->
	<node pkg="ar_sys" type="single_board" name="ar_single_board" output="screen">
		<remap from="/camera_info" to="/camera/rgb/camera_info" />
		<remap from="/image" to="/camera/rgb/image_rect_color" />
		
		<param name="marker_size" type="double" value="0.05"/>
		<param name="board_config" type="string" value="$(find ar_sys)/data/single/board.yml"/>
		<param name="board_frame" type="string" value="/board1" />
		<param name="draw_markers" type="bool" value="false" />
		<param name="draw_markers_cube" type="bool" value="true" />
		<param name="draw_markers_axis" type="bool" value="false" />
	</node>

	<!-- VIEW THE VISUALIZATION OF THE DETECTIONS -->
	<node pkg="image_view" type="image_view" name="image_view_rest" output="screen">
		<remap from="image" to="/ar_single_board/result" />
		<param name="autosize" type="bool" value="true" />
	</node>

</launch>